tests/compilation_cache_test.py

```diff
@@ -335,21 +337,23 @@ def test_cache_read_warning(self):
             "for 'jit__lambda_': RuntimeError: test error",
             str(w[0].message))
 
-  def test_min_instruction_count(self):
-    with tempfile.TemporaryDirectory() as tmpdir:
+  def test_min_compile_time(self):
+    with tempfile.TemporaryDirectory() as tmpdir, \
+         persistent_cache_min_compile_time_secs(2):
       cc.initialize_cache(tmpdir)
 
-      with persistent_cache_min_instruction_count(20):
-        # 2 instructions at time of writing
-        jit(lambda x: x * x)(2)
-        files_in_cache = len(os.listdir(tmpdir))
-        self.assertEqual(files_in_cache, 0)
+      # Takes between .02 and .08 seconds to compile as of Oct 28 2023
```

Comment:
I recommend against testing this way because the timing is very environment-dependent. If you have to do it, I would mock the time.

Reply:
Good idea, I mocked the time.